Tests/MongoSwiftSyncTests/ClientSessionTests.swift

```diff
@@ -1,8 +1,19 @@
 import Foundation
 @testable import MongoSwift
+@testable import MongoSwiftSync
```

Comment:
we need to import both modules here in order to access internal properties of async sessions e.g `id`. as a result a lot of the changes in this file are to help the compiler disambiguate between sync and async types.